safe/impact_functions/inundation/flood_vector_building_impact/impact_function.py

```diff
@@ -65,7 +65,9 @@ def notes(self):
                     'Buildings are said to be inundated when in a region with '
                     'field "%s" in "%s" .') % (
                         self.hazard_class_attribute,
-                        ', '.join(self.hazard_class_mapping[self.wet]))
+                        ', '.join([
+                            str(_) for _ in self.hazard_class_mapping[self.wet]
```

Comment:
is this unicode friendly? Also since the var is actually used it might be nice to call the ```_``` var ```hazard_class``` or something similar to make it more readable?